keen/explorer
# Fixes #117 Ensure table view is always 100% width.

## What does this PR do? How does it affect users?

It fixes the google charts table view and ensures it's always as wide as the entire visualization area. This makes sure data doesn't get cut off, like shown in #117.

## How should this be tested?

Try running a bunch of queries and using the table view. Confirm the table works as expected.

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:


## Related tickets?
#117
